### ISSUE TITLE
Opsdroid seems to default to apiai parser, on by default?

### ISSUE THREAD

**Issue by sgreszcz** [opsdroid/opsdroid#309]
# Description

Opsdroid seems to show in the debug logs that the apiai parser is being used, although it is commented out.  Is it on by default?  Also it doesn't show the parser that I would expect to be using (regex)

## Steps to Reproduce
Please also include relevant information and steps to reproduce the bug/issue.

Trigger a regex skill that responds with anything (for example text)

## Expected Functionality
Explain what should happen.

I'm assuming that the debug logs show the right parser being used, in this case regex?

## Experienced Functionality
Explain what happened instead(Please include the debug log).

But the debug shows, that it is using apiai, but really it should be using regex?:

DEBUG opsdroid.core: Parsing input: vcc load info
DEBUG opsdroid.core: Processing parsers
DEBUG opsdroid.core: Checking apiai

## Versions
- **Opsdroid version:**
opsdroid (0.9.1)
- **Python version:**
Python 3.5.2
- **OS/Docker version:**
Ubuntu 16.04.3

``` yaml
## Configuration File
Please include your version of the configuration file bellow.

## Parsers
parsers:
  - name: regex
    enabled: true
#
#   - name: crontab
#     enabled: true
#
#   - name: apiai
#     access-token: "youraccesstoken"
#     min-score: 0.6
```

## Additional Details
Any other details you wish to include such as screenshots, console messages, etc.

**Comment by jacobtomlinson** [comment]
This is a bug, the log message is not within the `if` statement which checks if apiai has been configured. This should be resolved in #311 and will be fixed in the next release.

**Comment by jacobtomlinson** [comment]
Should now be resolved in `v0.10.0`
